.gitmodules

```diff
@@ -217,5 +217,5 @@ path = src/dynamics/senh/dycore
 url = https://github.com/E3SM-Project/E3SM.git
 fxrequired = AlwaysRequired
 fxsparse = ../.e3sm_sparse_checkout
-fxtag = cec3bb6
+fxtag = 057df66
```

Comment:
Did you check the dycore files we have in cam_mods against the updates in this tag?

Reply:
Yes, it is tested and does not show any conflict yet.